basic-helm-spring-boot/.helm/spring-boot/templates/deployment.yaml

```diff
@@ -1,5 +1,5 @@
-apiVersion: apps/v1
-kind: Deployment
+apiVersion: apps.openshift.io/v1
+kind: DeploymentConfig
```

Comment:
We should continue to use `Deployment` object. The `image.openshift.io/triggers` annotation can be used to handle the triggering